update-docker-repo-info.py

```diff
@@ -22,6 +22,11 @@
 VERIFY_SSL = True
 
 DOCKERFILES_DIR = os.path.abspath(os.getenv('DOCKERFILES_DIR', '.dockerfiles'))
+try:
+    with open("docker_images_metadata.json", "r") as f:
+        DOCKER_IMAGES_METADATA = json.load(f)
+except Exception:
+    DOCKER_IMAGES_METADATA = {}
```

Comment:
perhaps worth checking only json parsing exception. and printing if the file isn't there or we failed to load the json.

Reply:
done